src/style/index.css

```diff
@@ -313,4 +313,42 @@ img {
 
 .service-view .owner .email {
 
+}
+
+.display-3 {
+  padding-top: 50px;
+  padding-bottom: 25px;
+}
+
+.avg-rating {
+  padding-top: 14px;
+  margin-left: 10px;
+  font-size: 15px;
+}
+
+.email {
+  margin-left: 10px;
+  font-size: 15px;
+  padding-top: 12px;
+}
+
+.counter {
+  padding: 10px;
+  width: 55px;
+  font-size: 25px;
+}
+
+hr {
```

Comment:
Please set this up so that it only affects the user page. It's also affecting the search on the home page